### DISCUSSION TITLE
Alternating Sound Output for AnalogAudioStream Using GeneratedSoundStream<int16_t> and AudioPlayer

### DISCUSSION THREAD

**Discussion by mitsuhito** [pschatzmann/arduino-audio-tools#1608] [Q&A] [answered]
I want to alternate the output sound for a single AnalogAudioStream using objects of GeneratedSoundStream<int16_t> and AudioPlayer. Can you provide an example or reference that could help me achieve this?

Thank you!

**Comment by pschatzmann** [comment, marked as answer]
If you are not able to figure this out yourself, maybe this is not the right tool for you!

The solution is just trivial: just set up both output chains both outputting to the same AnalogAudioStream and depending on the case you call the corresponding copy function.
